apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -979,10 +992,13 @@ object BftScanConnection {
         .sequentialTraverse(scans) { scan =>
           logger.info(s"Attempting to connect to Scan: $scan.")
           connectionBuilder(scan.publicUrl)
-            .transformWith { result =>
-              // logging
-              result.failed.foreach { err =>
-                // TODO(#815): abstract this pattern into the RetryProvider
+            .transformWith {
+              case Success(conn) =>
+                logger.info(
+                  s"Successfully connected to scan of ${scan.svName} (${scan.publicUrl})."
+                )
```

Comment:
Is this logged during every refresh? If that's the case i would actually lower this to debug. We should also lower the log above to debug in that scenario. And add a info log that logs all the scans that we're trying to connect to at the same time. And we will have the info logs for failures.